### PR TITLE
Added hot-reload

### PR DESCRIPTION
**Pull request by JulianWebb**
Also added a 'hot' script to 'package.json' for hot-reloads

~~Note that with current hot reload script, one now has to open the page in their browser themselves and refresh it. A hot reload script that can refresh the page while it's running without a complete restart would be preferred but I have no idea how to do that.~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "node ./build.js ./build",
     "serve": "node ./server.js ./build 3000",
-    "test": "npm run build && npm run serve"
+    "test": "npm run build && npm run serve",
+    "hot": "./node_modules/.bin/nodemon --exec npm test"
```
Comment (nkoder) [inline package.json:10]: just FYI: if package is available in `./node_modules/.bin` you can write tool name to achieve exactly same effect. In other words: `"hot": "nodemon --exec npm test"` does exactly same

File: server.js [hunk 2]
```diff
@@ -28,10 +28,5 @@ app.use(express.static(path.join(__dirname, BUILDDIR)));
 var server = app.listen(PORT, () => {
     console.log(`\nTemp server started at http://localhost:${PORT}!`);
     //console.log('press ctrl+c to stop ');
-
-    var opn = require('open');
```
Comment (nkoder) [inline server.js:32]: FYI: I will probably add another PR to bring back autoopen, this time as script run in parallel to `hot` script 👍

**Comment by nkoder** [comment]
> A hot reload script that can refresh the page while it's running without a complete restart would be preferred but I have no idea how to do that.

IMO this change already makes development way better 🚀

File: server.js (hunk 2 above)
Comment (JulianWebb) [reply server.js:32]: Resolved with commit 8fbe48e

File: package.json (hunk 1 above)
Comment (JulianWebb) [reply package.json]: Resolved with commit 8fbe48e

File: server.js (hunk 2 above)
Comment (JulianWebb) [reply server.js:32]: Should be addressed in commit 2771508

**Comment by JulianWebb** [comment]
This pull request is now a subset of #35

**Pull request closed without merging**